pymc3/sampling.py

```diff
@@ -27,7 +27,14 @@
 import sys
 sys.setrecursionlimit(10000)
 
-__all__ = ['sample', 'iter_sample', 'sample_ppc', 'sample_ppc_w', 'init_nuts', 'sample_prior_predictive']
+__all__ = [ 'sample',
+			'iter_sample',
```

Comment:
bad indent

Reply:
That indent doesn't look that way on my text editor.  I could just leave them all on the same line, but that would violate pep8.